app/cli/src/handlers/manifest/generator.ts

```diff
@@ -0,0 +1,102 @@
+import { uuid } from "@fsml/cli/deps/mod.ts";
+import { set } from "@fsml/cli/deps/lodash.ts";
+import { ManifestTypes } from "@fsml/cli/types/enums.ts";
+import { createValueForType, jsonToText, read } from "@fsml/packages/utils/mod.ts";
```

Comment:
it looks like this is violating the Deno linting rules

Reply:
Yes, odd that `deno lint` didnt catch it. `deno fmt` did. setting up a pre-commit would be nice. Not sure Ive yet found a good one for deno.